osf/metrics/reporters/spam_count.py

```diff
@@ -0,0 +1,76 @@
+from osf.models import OSFUser
+
+from osf.metrics.reports import SpamReport
+from ._base import MonthlyReporter
+from osf.models import PreprintLog, NodeLog
+from osf.models.spam import SpamStatus
+
+import dateutil
+from django.utils import timezone
+
+
+class SpamCountReporter(MonthlyReporter):
+
+    def report(self, report_date):
+        today = timezone.now()
+        last_month = (today - dateutil.relativedelta.relativedelta(months=1))
+
+        report = SpamReport(
+            report_date=report_date,
+            # Node Log entries
+            confirmed_spam_node=NodeLog.objects.filter(
+                action=NodeLog.CONFIRM_SPAM,
+                created__gt=last_month,
+                node__type='osf.node',
+            ).count(),
+            nodes_confirmed_ham=NodeLog.objects.filter(
+                action=NodeLog.CONFIRM_HAM,
+                created__gt=last_month,
```

Comment:
These should be `created__gte=target_month, created__lt=next_month` if using above recommendation